features/leaveroom/impl/src/main/kotlin/io/element/android/features/leaveroom/impl/LeaveRoomPresenterImpl.kt

```diff
@@ -85,6 +86,7 @@ private suspend fun showLeaveRoomAlert(
 ) {
     matrixClient.getRoom(roomId)?.use { room ->
         confirmation.value = when {
+            room.isDm -> Dm(roomId)
```

Comment:
As discussed with @giomfo IRL today, there is another case that we will need to handle: when the last admin in leaving the room, we should warn the user.